src/common/helpers/email-content/email-content.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {object} payload
```

Comment:
Similarly, more specific typing around `Application` would be beneficial here.

We assume later that `payload.sections` exists, but your autocompletion and other language server properties won't give you any help unless they know the specific incoming type.

Reply:
done